From cee3c3ab0817783e85f150ad5120d2ce9860a124 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=A4rkl?= <info@florianmaerkl.de>
Date: Fri, 9 May 2025 18:27:51 +0200
Subject: [PATCH] Remove unnecessary gio/gdesktopappinfo.h include

This was introduced in 5d7fbff4c97779922efe03268deaf669a5a4eaa8 but the
code using it was removed in 0312ffa2d621b398462659429a7c2b778da93ca8 so
it is not necessary anymore now. Moreover, this header is not
available when glib is compiled with cocoa so it would break the build
in such cases.
---
 src/indicator-desktop-shortcuts.c | 1 -
 1 file changed, 1 deletion(-)

diff --git src/indicator-desktop-shortcuts.c src/indicator-desktop-shortcuts.c
index be1d632..99a687a 100644
--- src/indicator-desktop-shortcuts.c
+++ src/indicator-desktop-shortcuts.c
@@ -25,7 +25,6 @@ License along with this library. If not, see
 #include "config.h"
 #endif
 
-#include <gio/gdesktopappinfo.h>
 #include "indicator-desktop-shortcuts.h"
 
 #define ACTIONS_KEY               "Actions"
-- 
2.49.0

